conda/environment-test-3.10.yml

```diff
@@ -17,6 +25,7 @@ dependencies:
   - pyarrow>=5
   - scipy
   - typing_extensions
+  - llvm-openmp
```

Comment:
Does the gcc package not come with openmp support? Do we need to also pull llvm here?

Reply:
This is to align with the package [build and runtime dependency](https://github.com/jjwilke/legate.core/blob/add-cmake-pr/conda/conda-build/meta.yaml#L92), which @m3vaz said we're using because it's faster than the regular `openmp` package.

Reply:
@trxcllnt @manopapad So openmp support is a bit messy. There are several openmp packages and I want to make sure we're clear on which we're talking about.
`openmp` is a dead package, and ***nothing*** should point at it.
`llvm-openmp` is generally a drop-in replacement for the old `openmp` package
`libgomp` is the GNU openmp package

There are a couple of reasons to prefer `llvm-openmp` over `libgomp`:
1) For reasons, gcc will output symbols that work with either `libgomp` or `llvm-openmp`, but clang will only produce symbols that work with `llvm-openmp`. 
2) Last I checked, `llvm-openmp` provides a wider platform support matrix (including support for Mac)
3) `libgomp` is not fork-safe

Does that clarify the situation?
This may also help: https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp